tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Modules/TSSModule.java

```diff
@@ -0,0 +1,510 @@
+package com.web3auth.tkey.ThresholdKey.Modules;
+
+import android.util.Pair;
+
+import androidx.annotation.Nullable;
+
+import com.web3auth.tkey.RuntimeError;
+import com.web3auth.tkey.ThresholdKey.Common.Result;
+import com.web3auth.tkey.ThresholdKey.Common.ThresholdKeyCallback;
+import com.web3auth.tkey.ThresholdKey.ThresholdKey;
+
+import org.json.JSONObject;
+import org.torusresearch.fetchnodedetails.types.NodeDetails;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.torusresearch.torusutils.TorusUtils;
+import org.torusresearch.torusutils.types.TorusPublicKey;
+import org.torusresearch.torusutils.types.VerifierArgs;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+class GetTSSPubKeyResult {
+    public static class Point {
+        public String x;
+        public String y;
+
+        public String toFullAddr() {
+            return "04" + x + y;
+        }
+
+        public Point(String x, String y) {
+            this.x = x;
+            this.y = y;
+        }
+    }
+
+    public Point publicKey;
+    public List<BigInteger> nodeIndexes;
+
+    public GetTSSPubKeyResult(Point publicKey, List<BigInteger> nodeIndexes) {
```

Comment:
```suggestion
    public GetTSSPubKeyResult(Point publicKey, List<int> nodeIndexes) {
```

Reply:
Think the conversion for known integers received as bigintegers is bigintvariable.intValue()